cpp/ps0000/problem 0.cpp

```diff
@@ -1,14 +1,15 @@
-#include <iostream>
+#include <iostream> // 2/2
 // first mistake : @ ,, # not #
 // second mistake : "iostream.h" ,, should be iostream without .h and ""

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`